tests/phpunit/Integration/InterwikiDBIntegrationTest.php

```diff
@@ -30,6 +30,8 @@ class InterwikiDBIntegrationTest extends SMWIntegrationTestCase {
 
 	private $queryResultValidator;
 	private $queryParser;
+	private $semanticDataFactory;
+	private $pageCreator;
```

Comment:
Remember, if you are touching this anyway, you could add types for these properties. You don't have to, but you can. It's some extra work, since you need to make sure the types are right, otherwise we get new bugs.